k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/tls-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cortx-ssl-cert-cfgmap-{{ .Release.Namespace }}
```

Comment:
This should really be something like `{{ include "cortx-configmap.fullname" . }}-ssl`, but I didn't want to make even more changes than I already have.